src/extension/neoExpress/packageVersion.ts

```diff
@@ -0,0 +1,73 @@
+import tryFetchJson from "../util/tryFetchJson";
+
+export default class PackageVersion {
```

Comment:
Express does use pre-release tags such as 3.6.4-preview. These are not typically delivered via nuget, but they can be installed via our [build server feed](https://dev.azure.com/ngdenterprise/Build/_artifacts/feed/public). Should we also provide support for prerelease suffixes here?

Also, this is just standard semantic versioning. Is there an existing package we can use?

Reply:
I haven't checked for existing packages. I am pretty sure there is. Although this is pretty simple to write, I will add the preview support and I have some unit tests to cover this class as well, so it should be easy.

Reply:
Another thought, we can add a configuration setting, for example, "include preview release: true|false"; default to false. This way, the user can opt-in/out explicitly.